examples/lunr/lunr.js

```diff
@@ -77,13 +77,17 @@ document.addEventListener('click', (e) => {
   if(e.target.matches('.button-click')) {
 
     window.aa('click',{
+      eventName: "hit-clicked",
+      indexName: process.env.INDEX_NAME,
```

Comment:
We should document the "best" way to access all those information.

Reply:
issue filed [here](https://github.com/algolia/search-insights.js/issues/25)